### PR TITLE
Change footer layout

### PR DESCRIPTION
**Pull request by lfdebrux**
Ticket: https://trello.com/c/lXpy2vZH/142-3-change-footer-to-use-two-column-layout-in-digitalmarketplace-frontend-toolkit-and-all-frontend-apps

## Screenshots
after

http://localhost/user/login

### CODE REVIEW

**Comment by aliuk2012** [comment]
Any idea why the frontend had its own footer.scss? I seem to recall there being a slight difference. it might have been just been fixing linting issues between the two i.e adding missing semi-colons and code style.

File: app/assets/scss/application.scss [hunk 1]
```diff
@@ -39,7 +40,6 @@ $govuk-compatibility-govukelements: true;
 @import "node_modules/digitalmarketplace-govuk-frontend/govuk-frontend/all";
 
 // App specific styles
-@import "_footer.scss";
 @import "_text.scss";
 @import "_secondary-action-link.scss";
```
Comment (aliuk2012) [inline app/assets/scss/application.scss]: Can we remove this? We aren't using secondary-action-link in this repo

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -5,7 +5,6 @@
   directives to concatenate multiple Javascript files into one.
 */
 //= require ../../../node_modules/jquery/dist/jquery.js
-//= require ../../../node_modules/hogan.js/web/builds/3.0.2/hogan-3.0.2.js
 //= require ../../../node_modules/digitalmarketplace-frontend-toolkit/toolkit/javascripts/option-select.js
 //= require ../../../node_modules/digitalmarketplace-frontend-toolkit/toolkit/javascripts/support.js
 //= require ../../../node_modules/digitalmarketplace-frontend-toolkit/toolkit/javascripts/list-entry.js
```
Comment (aliuk2012) [inline app/assets/javascripts/application.js]: Could we also remove list-entry.js and list-entry.scss as we do not use it in this repo?

**Comment by lfdebrux** [comment]
> Any idea why the frontend had its own footer.scss? I seem to recall there being a slight difference. it might have been just been fixing linting issues between the two i.e adding missing semi-colons and code style.

There wasn't any difference, I checked with `diff` before removing.

File: app/assets/scss/application.scss (hunk 1 above)
Comment (lfdebrux) [reply app/assets/scss/application.scss]: I think so? This should be in a separate PR though

File: app/assets/javascripts/application.js (hunk 2 above)
Comment (lfdebrux) [reply app/assets/javascripts/application.js]: As above, I think this should be in a separate PR.

**Pull request merged**